main.go

```diff
@@ -183,8 +186,8 @@ func main() {
 
 	router := route.New()
 
-	RegisterWeb(router)
-	api.Register(router.WithPrefix("/api"))
+	RegisterWeb(router.WithPrefix(amURL.Path))
+	api.Register(router.WithPrefix(amURL.Path + "/api"))
```

Comment:
Hmm, we're usually just +-ing together pre-sanitized paths in Prometheus, including in your own https://github.com/prometheus/common/blob/master/route/route.go. Not that it matters much, but maybe we should keep it consistent?